src/lib/realizeCustomElements.ts

```diff
@@ -234,6 +234,10 @@ function resolveOptions(registry: ReadOnlyRegistry, registryProvider: RegistryPr
 	return options;
 }
 
+function getTransitionOptionFromProjector(element: Element): boolean {
+	return element.getAttribute('data-css-transitions') === 'true';
```

Comment:
Something like `element.hasAttribute('data-css-transitions')` and ignoring the value?